src/service/poc_mobile.proto

```diff
@@ -152,6 +152,17 @@ enum cell_type {
 }
 
 message data_transfer_session_req_v1 {
+    // data_transfer_event represents traffic usage happening on a hotspot, it is traffic quota or timer based
+    // it is securely created and signed by hotspot
+    data_transfer_event data_transfer_usage = 1;
+    // this decision is done by OCS like system
+    bool reward_cancelled = 2;
+    // Online Charging System is part of teleco stack which is able to make a decision about data transfer and reward
+    bytes ocs_pub_key = 3;
```

Comment:
rename to `pub_key` .. the comment explains who's pub key it is

Reply:
fixed